rules/jest.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
     'jest/no-conditional-expect': 'off',
     'jest/require-top-level-describe': 'off',
     'jest/prefer-called-with': 'off',
-    'jest/prefer-inline-snapshots': 'off',
```

Comment:
this got deprecated in 25.0.0: https://github.com/jest-community/eslint-plugin-jest/blob/main/CHANGELOG.md#2500-2021-10-10, https://github.com/jest-community/eslint-plugin-jest/issues/661, so I believe we can remove it from overrides